File: Java/LV1/이상한 문자 만들기.java
import java.util.*;

class Solution {
    public String solution(String s) {
        String answer = "";
        
        int counter = 0;
        
        for(int i=0;i<s.length();i++){
            
            if(s.charAt(i)==' '){
                counter=0;
                answer+=" ";
                continue;
            }
            
            if(counter%2==0){
                String k = s.toUpperCase();
                answer+=k.charAt(i);
                counter++;
            }    
            
            else{
                String k = s.toLowerCase();
                answer+=k.charAt(i);
                counter++;
            }
        }
        return answer;
    }
}

File: Lv.1/이상한 문자 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    
    int index=0;
    for(int i=0;i<s.length();i++)
    {
        if(s[i]==32)
        {
            index=0;
            answer+=s[i];
        }
        else
        {
            if(index%2==0)
            {
                answer+=toupper(s[i]);
            
            }
        
        
            else
            {
                answer+=tolower(s[i]);
            
            }
            index++;
        }
        
        
    }
    return answer;
}
